src/test/java/com/cloudbees/jenkins/plugins/docker_build_env/FunctionalTests.java

```diff
@@ -47,12 +47,12 @@ public void run_inside_pulled_container() throws Exception {
     @Test
     public void run_inside_built_container() throws Exception {
         FreeStyleProject project = jenkins.createFreeStyleProject();
-        project.setScm(new SingleFileSCM("Dockerfile", IOUtils.toString(getClass().getResourceAsStream("/Dockerfile"), "UTF-8")));
+        project.setScm(new SingleFileSCM("Dockerfile", "FROM ubuntu:14.04"));
```

Comment:
any reason for this change?

Reply:
Hi @michaelneale, thanks again for taking a look. I actually found that this test was failing. I wonder if something has changed in the test libraries / harness that might be causing this. When I run this test, it fails like so:

```
=== Starting run_inside_built_container(com.cloudbees.jenkins.plugins.docker_build_env.FunctionalTests)
Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 19.34 sec <<< FAILURE! - in com.cloudbees.jenkins.plugins.docker_build_env.FunctionalTests
run_inside_built_container(com.cloudbees.jenkins.plugins.docker_build_env.FunctionalTests)  Time elapsed: 0.657 sec  <<< FAILURE!
java.lang.AssertionError: unexpected build status; build log was:
------
Legacy code started this job.  No cause information is available
Building in workspace /tmp/hudson6910613243922714787test/workspace/test0
Staging Dockerfile
Your project is missing a Dockerfile
FATAL: Interrupted
ha:AAAAWB+LCAAAAAAAAP9b85aBtbiIQSmjNKU4P08vOT+vOD8nVc8DzHWtSE4tKMnMz/PLL0ldFVf2c+b/lb5MDAwVRQxSaBqcITRIIQMEMIIUFgAAckCEiWAAAAA=java.lang.RuntimeException: Interrupted
	at com.cloudbees.jenkins.plugins.docker_build_env.DockerBuildWrapper.setUp(DockerBuildWrapper.java:181)
	at hudson.model.Build$BuildExecution.doRun(Build.java:154)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:536)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:374)
Finished: FAILURE
```

Both this change and the removal of "$WORKSPACE/" below got it to pass for me. I think that this method needs to be annotated with @LocalData but I couldn't figure out where to put the Dockerfile. There is one in the repo, under src/test/resources, but it doesn't seem to be found when the test runs. https://github.com/jenkinsci/docker-custom-build-environment-plugin/blob/master/src/test/resources/Dockerfile

Reply:
I checked out an earlier revision and this test passed, so I'll see if I can hunt down the change that broke it.

Reply:
58c931ef3dfc1f21a30e4354bfabb4838486c382 broke it. Will follow up on this.